src/opm/input/eclipse/Schedule/Action/ActionAST.cpp

```diff
@@ -47,9 +47,11 @@ AST AST::serializeObject()
 }
 
 Action::Result AST::eval(const Action::Context& context) const {
-    if (this->condition)
+    if (this->condition) {
+        if (this->condition->empty())
+            return Action::Result(false);
         return this->condition->eval(context);
-    else
+    } else
         return Action::Result(false);
```

Comment:
Personally I'd probably combine these conditions to reduce nesting
```C++
if (this->condition && !this->condition->empty())
    return this->condition->eval(context);
else
    return Action::Result(false);
```
or possibly reversed for that matter
```C++
if (!this->condition || this->condition->empty())
    return Action::Result(false);
else
    return this->condition->eval(context);
```

Reply:
I have updated to use the second alternative.